test/surf-documents.Tests.ps1

```diff
@@ -0,0 +1,68 @@
+param(
+    [String]$ClientId,
+    [String]$ClientSecret
+)
+
+. "test/common.ps1"
+
+$documentsPath = (Join-Path $PSScriptRoot (Join-Path "documents" "classification"))
+
+function New-Document([Io.FileInfo]$document) {
+    Invoke-App create document $document 2>&1
+}
+
+function Get-Documents {
+    Invoke-App list documents 2>&1
+}
+
+function Remove-Document([Parameter(ValueFromPipeline=$true)]$documentId) {
+    Invoke-App delete document $documentId 2>&1
+}
+
+Describe "documents" {
+    BeforeAll {
+        Login-Surf
+    }
+
+    BeforeEach {
+        # Tidy up any leftover documents in the account
+        Remove-Document --all
+    }
+
+    It "should be created from a file" {
+        $documentPath = (Join-Path $documentsPath "document1.pdf")
+        New-Document $documentPath | Format-MultilineOutput | Should -Match @"
+Creating document...
+  File                                      ID                      Size   Type               SHA256\s*
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  ...ocuments/classification/document1.pdf  ......................  50248  PDF:ImagePlusText  650c75913be04fa0f790abdcaddae6c9093b1d575cffbed2e098eb0de0e1d4b1..
```

Comment:
Are these literal `.`s, or regex `.`s (i.e., matching any character)?

Reply:
They're regex `.`s, matching any character (since we can't guarantee the ID of the new document.